ese2021-project-scaffolding/scaffolding/backend/src/services/user.service.ts

```diff
@@ -76,4 +81,15 @@ export class UserService {
     public getAll(): Promise<User[]> {
         return User.findAll();
     }
+
+    private checkPassword(password: string): boolean {
+        const contLowercase = new RegExp(/(?=.*[a-z]){1,}/).test(password);
```

Comment:
Ich verstehe nicht, was der Positive Lookahead oder die Gruppe im Pattern machen. Würde dies reichen?
```suggestion
        const contLowercase = new RegExp(/.*[a-z]+.*/).test(password);
        const contCapital = new RegExp(/.*[A-Z]+.*/).test(password);
        const contNumber = new RegExp(/.*[0-9]+.*/).test(password);
        const contSpecChar = new RegExp(/.*[@#$%^&-+=()\\]+.*/).test(password);
```